src/firebase-app.ts

```diff
@@ -35,6 +35,12 @@ import {FirestoreService} from './firestore/firestore';
  */
 export type AppHook = (event: string, app: FirebaseApp) => void;
 
+/**
+ * Constant holding the enviromnet variable that holds the default config.
+ */
+export const FIREBASE_CONFIG_FILE_VAR: string = 'FIREBASE_CONFIG';
+
+export const FIREBASE_CONFIG_KEYS = ['databaseURL', 'projectId', 'storageBucket'];
```

Comment:
Can we move this closer to the definition of `AppOption`? That will make it a little easier to keep those 2 in sync.